src/messaging/dotnet/src/Client/Client/WebSocket/MessageRouterBuilderWebSocketExtensions.cs

```diff
@@ -1,37 +0,0 @@
-﻿// Morgan Stanley makes this available to you under the Apache License,
```

Comment:
Why did you delete this file (and move its WS-specific methods to the more general builder class)?

Reply:
It seemed to me as if the extensions were remnants of an earlier version as they are in the same project it seemed like extra complexity to separate them like this.
I can revert this change, I don't have strong feelings, I didn't encounter this code organizing style before.

Reply:
This pattern is everywhere in the .NET ecosystem, it improves composability and makes it easier to extract specific components into separate packages. Indeed, there was no indication that this was the case here, but please revert as the whole websocket protocol binding might (should?) get moved to its own package.

Reply:
Reverted/moved the websocket-specific overload to the extension class